apps/src/clientApi.js

```diff
@@ -1,38 +1,71 @@
-/* global dashboard */
 // TODO(dave): Merge with the client API in /shared.
```

Comment:
ah yes, please remove! I'm all for combining client libraries, but there does not appear to be one in shared any more: https://github.com/code-dot-org/code-dot-org/tree/staging/shared/js . 

There is one in NetSim which @islemaster may be best able to comment on whether it should be merged with this, but I wouldn't worry about it for this PR: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimApi.js